lib/jekyll-crds/filters/media_url_filter.rb

```diff
@@ -0,0 +1,18 @@
+module Jekyll
+  module MediaUrlFilter
```

Comment:
Maybe `UrlParamFilter`?

Reply:
I'm not convinced on this one one way or another. UrlParamFilter to me signals "give me something to stick on the end and I will" but this is more about whether or not the item being passed in contains something specific to bitmovin currently. So it's only affected by media. I could see us reusing this for other very specific applications to media. I vote to table this until we run into it later